src/Tizen.WebView/Tizen.WebView/ContextMenuEventArgs.cs

```diff
@@ -0,0 +1,494 @@
+        /// </summary>
+        Translate,
+        /// <summary>
+        /// Copy link data.
+        /// </summary>
+        CopyLinkData,
+        /// <summary>
+        /// If app want to add customized item, use enum value after BaseApplicationTag.
+        /// </summary>
+        BaseApplicationTag = 10000
+    }
+
+    /// <summary>
+    /// Arguments from the ContextMenuItem smart event callback.
+    /// This class provides the properties of Context Menu item.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class ContextMenuItemEventArgs : EventArgs
+    {
+        private IntPtr _menuItemHandle;
+
+        internal ContextMenuItemEventArgs(IntPtr handle)
+        {
+            _menuItemHandle = handle;
+        }
+
+        internal IntPtr GetHandle()
+        {
+            return _menuItemHandle;
+        }
+
+        /// <summary>
+        /// Gets Tag of the context menu item.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public ContextMenuItemTag ItemTag
+        {
+            get
+            {
+                return (ContextMenuItemTag) Interop.ChromiumEwk.ewk_context_menu_item_tag_get(_menuItemHandle);
+            }
+        }
+
+        internal static ContextMenuItemEventArgs CreateFromSmartEvent(IntPtr data, IntPtr obj, IntPtr info)
+        {
+            return new ContextMenuItemEventArgs(info);
+        }
+    }
+
+    /// <summary>
+    /// Arguments from the ContextMenu smart event callback.
+    /// This class provides the properties of Context Menu.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class ContextMenuEventArgs : EventArgs
+    {
+        private IntPtr _menuHandle;
+
+        internal ContextMenuEventArgs(IntPtr handle)
+        {
+            _menuHandle = handle;
+        }
+
+        /// <summary>
+        /// Gets the context menu items count.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public int ItemCount
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_context_menu_item_count(_menuHandle);
+            }
+        }
+
+        /// <summary>
+        /// Gets Nth item of the context menu.
+        /// </summary>
+        /// <param name="n"> To get item at index n.</param>
+        /// <returns>The context menu item.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public ContextMenuItemEventArgs GetItemAtIndex(int n)
+        {
+            IntPtr itemHandle = Interop.ChromiumEwk.ewk_context_menu_nth_item_get(_menuHandle, n);
+            ContextMenuItemEventArgs item = new ContextMenuItemEventArgs(itemHandle);
+            return item;
+        }
+
+        /// <summary>
+        /// Removes item from the context menu.
+        /// </summary>
+        /// <param name="item"> The context menu item to be removed.</param>
+        /// <since_tizen> 6 </since_tizen>
+        public void RemoveItem(ContextMenuItemEventArgs item)
```

Comment:
How to add ContextMenuItem?
You only provide remove item

Reply:
In fact, most of apps have being used only the API what removes item.
And we received a request for this API only.
Could we add the API what appends new item later?

Reply:
To add Appends new item at later, 
You need to re-design 

You should provide the way to create ContextMenuItem. 
And This API should change function signature like below
```suggestion
        public void RemoveItem(ContextMenuItem item)
```

Reply:
And, I do not want an incompleted API.